File: URI/concluidos/Javascript/1020.js
var input = require('fs').readFileSync('./dev/stdin/arquivo.txt', 'utf8');
var lines = input.split('\n');

var dias = +lines.shift();

while (dias > 29){
    if (dias >=365){
        console.log(`${Math.floor(dias/365)} ano(s)`);
        dias = dias%365;
    }else{
        console.log(`${Math.floor(dias/365)} ano(s)`);
    }

    if (dias >= 30){
        console.log(`${Math.floor(dias/30)} mes(es)`);
        dias = dias%30;
    }else{
        console.log(`${Math.floor(dias/30)} mes(es)`);
    }
}
console.log(`${dias} dia(s)`);

File: URI/concluidos/Python/1020.py
dias = int(input())

ano = 0
mes = 0
dia = 0

while dias > 0:
    if dias >= 365:
        dias = dias - 365
        ano = ano + 1
    elif dias >= 30:
        dias = dias - 30
        mes = mes + 1
    elif dias >= 1:
        dias = dias - 1
        dia = dia + 1

print('%0d ano(s)' % ano)
print('%0d mes(es)' % mes)
print('%0d dia(s)' % dia)